recipes/zephyr-7b-beta/dpo/config_qlora.yaml

```diff
@@ -1,12 +1,12 @@
 # Model arguments
 model_name_or_path: alignment-handbook/zephyr-7b-sft-qlora
-torch_dtype: float16
+torch_dtype: bfloat16
```

Comment:
I turns out that using `bfloat16` makes a non-trivial difference to downstream perf! cc @nathan-az :)